test/e2e/run_test.sh

```diff
@@ -246,15 +276,23 @@ if [[ -z "${SKIP_INSTALL_DEPENDENCIES:-}" ]]; then
     install_dependencies
 fi
 
-if [[ -z "${SKIP_SETUP:-}" ]]; then
-    setup_kind_cluster
+if [[ -z "${SKIP_SETUP:-}" ]] && [[ "$USE_EXISTING_CLUSTER" = false ]]; then
+    setup_kind_cluster "$NAME"
 fi
 
 if [[ -z "${SKIP_BUILD:-}" ]]; then
     setup_kind_registry
     build_and_push_images
 fi
 
+kubectl config use-context "$NAME"
+echo "Running tests in cluster '$NAME'"
+
+if [[ "$USE_EXISTING_CLUSTER" = false ]]; then
```

Comment:
This code block is not needed.

Reply:
I guess lines 288 & 289 are not included.

Reply:
> I guess lines 288 & 289 are not included.

Yes, just `USE_EXISTING_CLUSTER` related code.